app/controllers/idv/personal_key_controller.rb

```diff
@@ -22,20 +22,6 @@ def update
       redirect_to next_step
     end
 
-    # Remove this after the next deploy
-    def download
-      personal_key = user_session[:personal_key]
-
-      analytics.track_event(Analytics::IDV_PERSONAL_KEY_DOWNLOADED, success: personal_key.present?)
```

Comment:
🌶️ one downside to not having a server action is we no longer get the analytics

However, I still feel strongly enough that having this action on the server is a liability, so I am OK not having analytics if it means we can have a more bug-free frontend

Reply:
https://cm-jira.usa.gov/browse/LG-6111 so we can track this

Reply:
We have some precedent for front-end logging with #5875, which might be useful for this.

(Also commented as such on related ticket)